app/frontend/src/common/authenticate.js

```diff
@@ -46,7 +50,8 @@ export default {
           refreshToken,
           idToken }
         ).success((result) => {
-          store.commit(SET_KEYCLOAK, instance)
```

Comment:
Is there a reason we can't just use the imported values?

Reply:
The submissions app will have it's own store so this file can't import the registry's store.
This and another comment could be resolved by creating a store module for auth-related stuff (keycloak instance and roles) that all apps can include- I'll look into that.

We could also talk about storing the keycloak instance on the Vue instance and access it with `this.$keycloak`